decision_service/dsclientlib/ds_restapi_data_transport.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include "model_mgmt/ds_model_mgmt.h"
+
+namespace decision_service { namespace model_mangement {
+  class restapi_data_tranport : public i_data_transport {
+  public:
+    model_data get_data() override;
+  };
+}}
```

Comment:
Why is dsclientlib, a binary file, included in the pull request?

Reply:
Removed.  Yann will help me tomorrow from removing it from commit history.

---
In reply to: [186873323](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r186873323) [](ancestors = 186873323)